src/components/Button/__tests__/Button.visual.test.tsx

```diff
@@ -89,20 +89,36 @@ test.describe('Button', {tag: '@Button'}, () => {
         await expectScreenshot();
     });
 
-    const smokeScenarios = createSmokeScenarios(defaultProps, {
-        size: sizeCases,
-        selected: selectedCases,
-        disabled: disabledCases,
-        loading: loadingCases,
-        view: viewsCases,
-        pin: pinsCases,
-    });
-
-    smokeScenarios.forEach(([title, details, props]) => {
+    const qa = 'test-button';
+
+    createSmokeScenarios<ButtonProps>(
+        {
+            children: 'Text',
+            qa,
+        },
+        {
+            size: sizeCases,
+            selected: selectedCases,
+            disabled: disabledCases,
+            loading: loadingCases,
+            view: viewsCases,
+            pin: pinsCases,
+        },
+    ).forEach(([title, details, props]) => {
         test(title, details, async ({mount, expectScreenshot}) => {
-            await mount(<Button {...props} />);
+            const root = await mount(<Button {...props} />);
 
             await expectScreenshot();
+
+            if (props.disabled || props.loading) {
+                return;
+            }
+
+            await root.getByTestId(qa).hover();
+
+            await expectScreenshot({
+                screenshotPostfix: 'after hover',
```

Comment:
Let's use `hovered`, thus screenshot name will be more readable: `Button-smoke-hovered-...`

Reply:
renamed